src/lib.rs

```diff
@@ -8,6 +8,9 @@ and allows you to create to create timer scopes with minimal overhead!
 # How to use
 
 ```
+
+#![warn(clippy::doc_markdown)]
```

Comment:
why change the warning level for this here and not configure it globally in `Cargo.toml`?